src/components/filters/DateFilter/DateFilterBody/tests/DateFilterBody.spec.tsx

```diff
@@ -64,4 +64,42 @@ describe("ExtendedDateFilterBody", () => {
         const rendered = shallow(<DateFilterBody {...mockProps} isEditMode={false} />);
         expect(rendered.find(EditModeMessage)).not.toExist();
     });
+
+    describe("calculateHeight", () => {
+        const { innerWidth, innerHeight } = window;
+        const resizeWindow = (x: number, y: number) => {
+            (window.innerWidth as number) = x;
+            (window.innerHeight as number) = y;
+        };
+
+        const getBodyWrapper = (component: ShallowWrapper) => {
+            return component.find(".gd-extended-date-filter-body-wrapper");
+        };
+
+        const getBodyScroller = (component: ShallowWrapper) => {
+            return component.find(".gd-extended-date-filter-body-scrollable");
+        };
+
+        afterAll(() => {
```

Comment:
Please help me check this `afterAll` is neccessary?
IMO, it's not.

Reply:
removed.
thanks :)